modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
-= Cluster Configuration Properties
-:description: Cluster configuration properties list.
+= Cluster Configuration Properties 
+:description: Cluster configuration properties list. 
 
 Cluster configuration properties are the same for all brokers in a cluster. They can be set at the cluster level.
 
-For information on how to edit cluster properties, see xref:manage:cluster-maintenance/cluster-property-configuration.adoc[Configuring Cluster Properties]. Some properties require the cluster to be restarted in order to be applied; see a specific property's reference for whether restart is required.
+For information on how to edit cluster properties, see xref:manage:cluster-maintenance/cluster-property-configuration.adoc[]. Some properties require the cluster to be restarted in order to be applied; see a specific property's reference for whether restart is required.
 
----
-
-== Admin properties
-
-=== admin_api_require_auth
+== Cluster Configuration
```

Comment:
```suggestion
== Cluster configuration
```

Reply:
@Deflaimun the other pages just use Broker or Cloud (not Broker configuration or Cloud configuration). All pages should be consistent.